src/sybil/row_store.go

```diff
@@ -129,20 +129,20 @@ func (t *Table) LoadRecordsFromLog(filename string) RecordList {
 	ret := make(RecordList, len(marshalledRecords))
 
 	for i, r := range marshalledRecords {
-		ret[i] = r.toRecord(t)
+		ret[i] = r.toRecord(flags, t)
 	}
 	return ret
 
 }
 
-func (t *Table) AppendRecordsToLog(records RecordList, blockname string) {
+func (t *Table) AppendRecordsToLog(flags *FlagDefs, records RecordList, blockname string) {
 	if len(records) == 0 {
 		return
 	}
 
 	// TODO: fix this up, so that we don't
```

Comment:
so that we don't what?! damnit, past me - what warning were you trying to give us?